docs/HowToFAQ/Errors.md

```diff
@@ -10,8 +10,11 @@ layout: default
 
 <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
 
+This page contains a catalog of the dafny tool's Error and Warning messages,
```

Comment:
```suggestion
This page contains a catalog of Dafny's error and warning messages,
```

Reply:
Edited, but left 'dafny' as that is the tool name.

Reply:
That's fine with me